### ISSUE TITLE
upgraded to latest version--admin center page doesn't load

### ISSUE THREAD

**Issue by smcclements** [pnp/custom-learning-office-365#156]
> Thank you for reporting an issue or suggesting an enhancement. We appreciate your feedback - to help the team to understand your needs, please complete the below template to ensure we have the necessary details to assist you.
>
> 
>

#### Category

>**PROVISIONING NOTE**: If you are having issues with the provisioning service while trying to provision M365 learning pathways please do not post that issue to this repo as we cannot help you here. Please post all questions and issues for the provisioning service to [https://github.com/SharePoint/sp-dev-docs/issues](https://github.com/SharePoint/sp-dev-docs/issues).

- [x] Question
- [ ] Bug
- [ ] Content

Uploaded to the new version of web part 3.0.1.0.  When I click on the Custom Learning Admin page it only show the playlist not the page.

#### Question / Expected or Desired Behavior

Uploaded to the current web part version

#### Observed Behavior

Click on Administration Center page and it just displays the playlist.  Also I noticed that all the pages in the site contents still have the date I provisioned the site 9/27/10

#### Steps to Reproduce

Went to app catalog and uploaded the latest version of web part. Went to the site, checked site contents and clicked on the app part and updated to most recent version.  Reloaded the site.

#### Submission Guidelines

Thanks for your contribution! Sharing is caring.

**Comment by juliemturner** [comment]
Thank you for submitting something but I'm a little lost, did you not fill in the issue with any details? I'm not seeing anything in here, just a title that the "admin center page doesn't load". It's very hard for me to assist you if you don't provide any details. If you could you please edit the issue and fill in the issue template, removing the sections that are instructional and providing some better detail about what you're experiencing I will endeavor to assist you.

**Comment by smcclements** [comment]
Sorry I didn't read the template correctly. I have edited it now.

**Comment by juliemturner** [comment]
Ok, a couple of things.
1. Updating the web part does only that, updates the admin and viewer web parts, not the actual pages themselves so no the modified dates of the pages would not change. In this case the update gives new functionality and bug fixes to the admin web part and the viewer web part. In addition, a content update was recently released. This too is separate from the web part and the pages and comes to you automatically without doing any real interaction.
2. When you say you click on the "Admin Center Page" and it only displays a playlist that seems odd, can you please confirm that the "admin" page you're referencing is the page "CustomLearningAdmin.aspx"? If so then can you please share a screen shot of the page so I can understand exactly what you mean by "..just displays the playlist".

Thanks.

**Comment by smcclements** [comment]
![Admin Success Center](https://user-images.githubusercontent.com/57961998/69180726-c35d2b00-0adb-11ea-91d4-f4ddbed6873f.jpg)

**Comment by smcclements** [comment]
Do I need to do anything to update the content?  I just uploaded the web part update.

**Comment by juliemturner** [comment]
*TL;DR;* Content updates happen automatically with no intervention; web part updates are manual.

The web part and the content are completely separate things. I don't know what version you were updating from so I can't speak to what you were seeing before, but updating the web part does not update the content, however, since we're so close to Ignite and that's when the new content was released it's sort of showing you both at one time.

What the screenshot is showing me is a new sub-category of the category **Adoption Tools** called *Admin Success Center* which had 5 new playlists in it.  I don't actually see any issue, if you click on the categories and sub-categories on the left nav you will see the listings of all the playlists now offered.

**Comment by smcclements** [comment]
I thought the pag
![Admin Success Center](https://user-images.githubusercontent.com/57961998/69183980-f4d8f500-0ae1-11ea-9df9-65828e3ccabf.jpg)
e would look like below

**Comment by juliemturner** [comment]
Maybe we're having a terminology issue here.  That URL is displaying a page, and the page does not have a playlist on it.  The page has the admin view web part on it that allows you to administer the content for learning pathways. If you're asking me where you go to view the contents of a playlist there's another page automatically created in the library called `CustomLearningViewer.aspx` that allows you to navigate the content.  Maybe that's what you're looking for.

**Comment by smcclements** [comment]
Thank you--I think that was my issue.  I will watch that demo too.
